storage/local/storage.go

```diff
@@ -360,11 +371,28 @@ func (s *memorySeriesStorage) GetMetricForFingerprint(fp clientmodel.Fingerprint
 
 // AppendSamples implements Storage.
 func (s *memorySeriesStorage) AppendSamples(samples clientmodel.Samples) {
+	if len(samples) == 0 {
```

Comment:
But let's still check for emptiness, just for robustness. It's so fast compared to everything else in that method.